### DISCUSSION TITLE
Expected histogram behavior?

### DISCUSSION THREAD

**Discussion by RobertM314** [JetBrains/lets-plot#1160] [General]
I apologize if I'm posting this in the wrong location.

I'm confused about what's happening when using geom_histogram(aes(y='..density..')

```
import numpy as np
import pandas as pd
from lets_plot import *
LetsPlot.setup_html()

rng = np.random.default_rng(12345)

test = rng.normal(0, 1, (20, 2))
df = pd.DataFrame(test, columns=['A', 'B'])

plot_1 = (
    ggplot(df, aes(x='A')) +
    geom_histogram(binwidth=.2, fill='pink') +
    ggtitle('Frequency')
)
plot_1.show()

plot_2 = (
    ggplot(df, aes(x='A')) +
    geom_histogram(aes(y='..density..'),binwidth=.2, fill='pink') +
    ggtitle('Density')
)
plot_2.show()

plot_3 = (
    ggplot(df, aes(x='A')) +
    geom_histogram(aes(weight=np.ones_like(df.A) / len(df.A)),binwidth=.2, fill='pink') +
    ggtitle('Probability')
)
plot_3.show()
```
Plot 1 makes sense to me, as does plot 3, but plot 2 seems wrong. For example, the frequency in the first bin is 3, there are a total of 20 data points, the bin width is .2, I would expect the density in the first bin to be:
$\frac{(\frac{3}{20})}{.2} = .75$
Rather than the .84 indicated on the plot. 
Is my understanding of the '..density..' aesthetic faulty, and if so, what is it actually doing?

Thanks in advance.

**Comment by ASmirnov-HORIS** [comment]
Hello!
Yes, you are absolutely right! There is an error in the calculation of `..density..`.
Please see issue #1157.
